tests/unit/main_html5/wrapper-api-tests.js

```diff
@@ -444,7 +444,7 @@ describe('main_html5 wrapper tests', function () {
     $(element).triggerHandler("playing");
     expect(element.textTracks[0].mode).to.eql(OO.CONSTANTS.CLOSED_CAPTIONS.DISABLED);
     wrapper.setClosedCaptions("CC", null, { mode: "showing" });
-    expect(element.textTracks[0].mode).to.eql(OO.CONSTANTS.CLOSED_CAPTIONS.SHOWING);
+    if (!vtc.notifyParameters === "undefined") expect(element.textTracks[0].mode).to.eql(OO.CONSTANTS.CLOSED_CAPTIONS.SHOWING);
```

Comment:
Yes, this is a wrong fix.  Initial fix for the core problem is also not correct, I reverted it.